sdk/src/Services/DynamoDBv2/Custom/DataModel/_async/AsyncSearch.IAsyncEnumerable.cs

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright 2012-2013 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Remove the year. Just say "Copyright Amazon.com, ..."

Reply:
Thank you. I fixed this.